common/src/traversal.rs

```diff
@@ -11,8 +11,10 @@ use crate::{
     proto::Position,
 };
 
-/// Ensure all nodes within `distance` of `start` exist
+/// Ensure all nodes exist whose bounding spheres are within `distance` of `start`
```

Comment:
We lost the word "exist" here.

Reply:
Wait, no we didn't.

Reply:
Oh yeah, the grammar feels a bit weird, but I thought it would be easier to read, since a sentence ending in "`start` exist" might end up being a garden path sentence.